engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -202,6 +208,7 @@ class EngineJobExecutionActor(replyTo: ActorRef,
   onTransition {
     case fromState -> toState =>
       log.debug("Transitioning from {}({}) to {}({})", fromState, stateData, toState, nextStateData)
+      eventList :+= ExecutionEvent(toState.toString)
```

Comment:
Yup, they're mixed into the `eventList` when the EJEA gets a `SucceededResponse`